Fix specification of dependency version

AndiDog opened the pull request:
Was always incorrect, and from Rust 1.9 or so, Cargo will actually warn about it (will become an error at some point).

It must either be

``` toml
[dependencies.time]
version = "^0.1.5"
```

or just plain below "[dependencies]":

``` toml
[dependencies]
time = "^0.1.5"
```

Commit:
Fix specification of dependency version

dckc commented:
Any idea why the travis-ci build is failing? It's not obvious to me from a quick look at the build log.

AndiDog commented:
The failing tests from the Travis build work for me, but I get a single failure

> test _0 ... FAILED
> 
> failures:
> 
> ---- _0 stdout ----
>         thread '_0' panicked at 'test executable failed:
> 
> thread '<main>' panicked at 'retrieving column 2: Time parsing error: Invalid year. (20)', src\lib.rs:309

... which makes sense just by looking at it. Probably that's from using another version of `time`? I will check soon what the problem is.

AndiDog committed:
Fix another bad memory access

AndiDog commented:
I rebased onto my memory access changes, fixed another bad memory access and now it builds fine. Also on my Windows machine where the memory problem would typically occur immediately.
